server/utils/constants.js

```diff
@@ -835,6 +836,11 @@ const JOB_ERROR_TYPES = {
   UNKNOWN_ERROR: 'unknown-error',
 };
 
+const CLOCK_TYPES = {
+  ANALOG: 'analog',
+  DIGITAL: 'digital',
+};
+
```

Comment:
Is it something that really needs to be in the server constants? It seems it's something front only right?

It could just be in the components in the front imo :)

Reply:
Yes, it's not used in backend part, I have moved this constant into component